src/components/pathway/SearchPathwayCard.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useContext, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import Truncate from 'react-truncate';
+import { Link } from 'react-router-dom';
+import Skeleton from 'react-loading-skeleton';
+import { AppContext } from '@edx/frontend-platform/react';
+import { getConfig } from '@edx/frontend-platform/config';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import { Badge, Card } from '@edx/paragon';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import classNames from 'classnames';
+
+const filterTags = tags => {
```

Comment:
nit: can we add some comments about this / in this function?

Reply:
Also can we rename it to something more specific, like e:g "filterSkills"

Reply:
I have added comments explaining the functionality and the reason I used `tag` instead of `skill` is because same term was used in the ticket and I wanted to keep this parameter generic.